code/datums/elements/kneejerk.dm

```diff
@@ -0,0 +1,76 @@
+/// An element which enables certain items to tap people on their knees to measure brain health
+/datum/element/kneejerk
+	element_flags = ELEMENT_BESPOKE | ELEMENT_DETACH
+	id_arg_index = 2
```

Comment:
There's no args, so this entire thing should just be `element_flags = ELEMENT_DETACH`.